index.html

```diff
@@ -542,6 +542,42 @@ <h2 id="constrainable-properties">
             </tr>
           </tbody>
         </table>
+        <p>The following new and existing <code><a>MediaStreamTrack</a></code>
+        <a href="https://www.w3.org/TR/mediacapture-streams/#constrainable-properties">
+        Constrainable Properties</a> are defined to apply to the user-selected
+        audio sources, with the following behavior:</p>
+        <table class="simple">
+          <thead>
+            <tr>
+              <th>Property Name</th>
+              <th>Type</th>
+              <th>Behavior</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr id="def-constraint-width">
+              <td><dfn>ownAudioCancellation</dfn></td>
+              <td><code><a href="https://www.w3.org/TR/mediacapture-streams#dom-constrainboolean">ConstrainBoolean</a></code></td>
+              <td>
+                <p>As a setting, this value indicates whether or not the user
+                agent is applying <a>own audio cancellation</a> to the
+                source.</p>
+                <p>As a constraint, this property can be constrained to the
+                value <code>true</code>, resulting in a source that has
+                <a>own audio cancellation</a> applied.</p>
+                <p>When <dfn>own audio cancellation</dfn> is applied, the user
+                agent MUST attempt to remove any audio from the audio being
+                captured that was produced by the application that performed
+                <code>getDisplayMedia</code>. If the user agent is not able to
+                remove the audio through processing it SHOULD remove the audio
+                by excluding the application's audio from being captured. If
+                this results in no audio being captured, the user agent MUST
+                keep the track muted until it is able to capture audio
+                again.</p>
```

Comment:
"application" here is very vague.  Does it mean:

1. the browser
2. the specific browser process
3. the document origin
4. the document
5. the window/tab

Relevant questions I'd like to be able to answer:

* if capture is initiated from an iframe, does the audio from the parent get captured?
* similarly, for a parent, does the audio from an iframe get captured?
* if the same origin is playing audio from multiple tabs, is it only the tab that calls this API that is suppressed, or are all of these expected to be suppressed?

I realize that there are very few cases where audio comes from multiple sources and the primary use of this will be for origins that want to ensure that their own output, coming from the same page, is not echoed back.  But this does need to be clear.

Reply:
I should have requested changes.  It's not great to merge a PR when it is so vague, unless it is making progress on an outstanding issue.  This is adding a feature, so it could have waited.

Reply:
I would normally agree, but I think it was understood by all the editors that it meant the "same JavaScript" that performed getDisplayMedia(), for example "tab" or "document" and not "browser". An "application" here is not something that is shared between tabs from the same origin, and this can be clarified. What happens in edge cases like when iframes are used, or what precise wording best nails down these nuances, should definitely be discussed, but I don't think this should block us from adding the constraint in the first place. Chrome is already looking into adding audio and I want to make sure this constraint is outlined sooner rather than later as a heads-up of the direction this is going, since the issue in question #79 is a long and complicated one, and not knowing about the possibility of such constraint might take you down wrong turns. Let's iterate on clarifying "application" at #102.